node/src/BaseClient.ts

```diff
@@ -1660,8 +1660,12 @@ export class BaseClient {
      * console.log(result); // Output: ["field1", "field2", "field3"]  - Returns all the field names stored in the hash "my_hash".
      * ```
      */
-    public async hkeys(key: string): Promise<string[]> {
-        return this.createWritePromise(createHKeys(key));
+
+    public async hkeys(
+        key: GlideString,
+        decoder?: Decoder,
```

Comment:
We are using decoder option as of now
```suggestion
        options?: DecoderOption,
```

Reply:
Add a corresponding doc for it

Reply:
You need to update tests too